{
  "comments": [
    {
      "key": {
        "uuid": "3a79b1b5_1d7c1772",
        "filename": "dci/server/api/v1/componenttypes.py",
        "patchSetId": 5
      },
      "lineNbr": 37,
      "author": {
        "id": 30
      },
      "writtenOn": "2015-10-29T12:52:31Z",
      "side": 1,
      "message": "I understood you were willing to drop the uuid.UUID() instance and use standard string instead.",
      "revId": "6333601f4d7495c75acfafa619c551458301fa8e",
      "serverId": "6c1dc8ef-8b94-40e4-bd83-c2359d45ecc0",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "3a79b1b5_7d874b62",
        "filename": "dci/server/api/v1/componenttypes.py",
        "patchSetId": 5
      },
      "lineNbr": 53,
      "author": {
        "id": 30
      },
      "writtenOn": "2015-10-29T12:52:31Z",
      "side": 1,
      "message": "the constraint error may be generated by another error. I think we should preserve the exception and show it in the error message for reference. This at last until we are able to log them properly.",
      "revId": "6333601f4d7495c75acfafa619c551458301fa8e",
      "serverId": "6c1dc8ef-8b94-40e4-bd83-c2359d45ecc0",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "3a79b1b5_fdd91b84",
        "filename": "dci/server/api/v1/componenttypes.py",
        "patchSetId": 5
      },
      "lineNbr": 53,
      "author": {
        "id": 29
      },
      "writtenOn": "2015-10-29T13:18:41Z",
      "side": 1,
      "message": "Done",
      "parentUuid": "3a79b1b5_7d874b62",
      "revId": "6333601f4d7495c75acfafa619c551458301fa8e",
      "serverId": "6c1dc8ef-8b94-40e4-bd83-c2359d45ecc0",
      "unresolved": false
    }
  ]
}